test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,10 +94,11 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        if (command.isMutating()) {
```

Comment:
The test is incomplete. How should you go about testing the case where the command is not mutating?